root/static/scripts/release-editor/edits.js

```diff
@@ -195,8 +197,10 @@ releaseEditor.edits = {
             newMediumData = _.cloneDeep(newMediumData);
 
             if (medium.id) {
-                var newNoPosition = _.omit(newMediumData, "position");
-                var oldNoPosition = _.omit(oldMediumData, "position");
+                const newNoPosition = {...newMediumData};
```

Comment:
Just me being weird, but maybe change these to newWithoutPosition or something? I was wondering what number "No" referred to :D